cstore_fdw.c

```diff
@@ -358,7 +361,7 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 								  destReceiver, completionTag);
 		}
 
-		TruncateCStoreTables(cstoreTablesList);
+		TruncateCStoreTables(cstoreRelationList);
```

Comment:
`heap_close()` the relations after truncation? (See `ExecuteTruncate()` in tablecmds.c)